arch/arm/src/stm32f7/stm32_bbsram.c

```diff
@@ -424,7 +424,7 @@ static off_t stm32_bbsram_seek(struct file *filep, off_t offset,
    *   point, subsequent reads of data in the gap shall return bytes with the
    *   value 0 until data is actually written into the gap."
    *
-   * We can conform to the first part, but not the second.  But return EINVAL
+   * We can conform to the first part, but not the second. But return -EINVAL
```

Comment:
```suggestion
   * We can conform to the first part, but not the second.  But return -EINVAL
```
I think most of the comments have 2 spaces after dot

Reply:
Yes, but it make the length > 79char/line

Reply:
yeah, but newline is an option :)